unitylibs/scripts/utils.js

```diff
@@ -64,6 +64,14 @@ export function defineDeviceByScreenSize() {
   return 'TABLET';
 }
 
+export function getLocale() {
+  const urlLocale = window.location.pathname.split('/')[1];
+  const config = getConfig();
+  const { locales } = config;
```

Comment:
Nit: this can be simplified
```suggestion
  const { locales } = getConfig();
```